zuul.d/content_provider.yaml

```diff
@@ -16,18 +16,29 @@
       - ^kuttl-test.yaml$
     required-projects:
       - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/ci-framework
```

Comment:
Please add https://github.com/openstack-k8s-operators/telemetry-operator to this list also. sorry missed in the last comment.

Reply:
@raukadah this one is not in `zuul.d/rdo.yml`. Isn't going to cause an outage?

Reply:
Indeed, I had to add https://review.rdoproject.org/r/c/config/+/49761 to allow for `telemetry-operator` to be imported here.